tests/framework/framework.go

```diff
@@ -229,13 +232,17 @@ func (f *Framework) GetKubeClient() (*kubernetes.Clientset, error) {
 }
 
 func GetKubeClientFromFlags(master, kubeconfig string) (*kubernetes.Clientset, error) {
-	config, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	config, err := LoadConfig()
 	if err != nil {
 		return nil, err
 	}
 	return GetKubeClientFromRESTConfig(config)
 }
 
+func LoadConfig() (*rest.Config, error) {
+	return clientcmd.BuildConfigFromFlags(*master, *kubeConfig)
```

Comment:
don't refer to globals when these ptrs are passed to `GetKubeClientFromFlags`. Just need to pass them to `LoadConfig` too.

Reply:
Fixed